bigtop_toolchain/manifests/packages.pp

```diff
@@ -29,6 +29,8 @@
         "gcc",
         "gcc-c++",
         "fuse",
+        "fuse3",
+        "fuse3-devel",
```

Comment:
While this affects all Red Hat clones, I'm ok here since fuse3 and fuse3-devel is available in CentOS 7 and Rocky Linux 8 at least.